src/hotspot/cpu/x86/x86.ad

```diff
@@ -1896,6 +1896,12 @@ const bool Matcher::match_rule_supported_vector_masked(int opcode, int vlen, Bas
     case Op_FmaVD:
       return true;
 
+    case Op_MacroLogicV:
+      if(bt != T_INT && bt != T_LONG) {
```

Comment:
Missing `VM_Version::supports_evex()` check?

Reply:
Hi @vnkozlov, we already have that check (UseAVX < 3) in match_rule_supported routine which gets called from this function.

Reply:
Good.